mod/ping.php

```diff
@@ -477,7 +478,11 @@ function ping_get_notifications($uid)
 
 			if ($notification["visible"] && !$notification["spam"] &&
 				!$notification["deleted"] && !is_array($result[$notification["parent"]])) {
-				$result[$notification["parent"]] = $notification;
+				if (PConfig::get(local_user(), 'system', 'detailed_notif')) {
```

Comment:
Can you please add a comment that this part is for showing comments + like notifs (detailed_notif is not that self explaining. So a comment would be useful)

Reply:
Yeah, you are right.